website/source/docs/connect/registration/service-registration.html.md

```diff
@@ -185,6 +190,9 @@ followed by documentation for each attribute.
   reference](/docs/connect/configuration.html#envoy-options)
 * `mesh_gateway` `(object: {})` - Specifies the mesh gateway configuration
    for this proxy. The format is defined in the [Mesh Gateway Configuration Reference](#mesh-gateway-configuration-reference).
+* `expose` `(object: {})` - Specifies the configuration to expose HTTP paths through this proxy. 
```

Comment:
Is `expose` an upstream configuration? We might want to add it to the example if it should go here.

Reply:
It's not, `expose` is only relevant to the local proxy.